### PR TITLE
Add links and QR codes to builds

### PR DESCRIPTION
**Pull request by cobbal**
refs: MBL-13958
affects: none
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
## Installable Builds
|Student|Teacher|Parent|
|:-----:|:-----:|:----:|
|[qr code for student](rebrand.ly/eg2r4iw)|[qr code for teacher](rebrand.ly/jo6qa50)|[qr code for parent](rebrand.ly/ryc4bkr)|

**Comment by inst-danger** [comment]
## Installable Builds
|Student|Teacher|Parent|
|:-----:|:-----:|:----:|
|[qr code for student)](rebrand.ly/vv6t0kc)|[qr code for teacher)](rebrand.ly/me284m5)|[qr code for parent)](rebrand.ly/6srsl6h)|

**Comment by inst-danger** [comment]
## Installable Builds
|Student|Teacher|Parent|
|:-----:|:-----:|:----:|
|[qr code for student](rebrand.ly/z6vld39)|[qr code for teacher](rebrand.ly/86aspta)|[qr code for parent](rebrand.ly/1dku458)|

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Please add a description for Danger.
  </td>
    </tr>
  </tbody>
</table>



Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.63% | 93.63% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by inst-danger** [comment]
## Installable Builds
student: [link](rebrand.ly/z6vld39) [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fz6vld39)
teacher: [link](rebrand.ly/86aspta) [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F86aspta)
parent: [link](rebrand.ly/1dku458) [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F1dku458)

**Comment by inst-danger** [comment]
## Installable Builds
student: [Link](https://rebrand.ly/z6vld39) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fz6vld39)
teacher: [Link](https://rebrand.ly/86aspta) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F86aspta)
parent: [Link](https://rebrand.ly/1dku458) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F1dku458)

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/cmf3gjn) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fcmf3gjn)
[qr code for student](https://rebrand.ly/cmf3gjn)
teacher: [Link](https://rebrand.ly/l20k1jm) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fl20k1jm)
[qr code for teacher](https://rebrand.ly/l20k1jm)
parent: [Link](https://rebrand.ly/m4zmrp1) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fm4zmrp1)
[qr code for parent](https://rebrand.ly/m4zmrp1)

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/ukshqs6) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fukshqs6)
teacher: [Link](https://rebrand.ly/ztvhwh6) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fztvhwh6)
parent: [Link](https://rebrand.ly/yyajrqd) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fyyajrqd)

File: scripts/build_automation/build-link.swift [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+        return
+    }
+    try Rebrandly.shortenOrUpdate(id: ids[appIndex], url: url, title: app.title(forPr: prID))
+}
+
+let usage = """
+usage:
+    ./scripts/build_automation/build-link.swift generate-temp-links PR_NUMBER
+    ./scripts/build_automation/build-link.swift update-link student|teacher|parent BRANCH URL
+"""
```
Comment (cobbal) [inline scripts/build_automation/build-link.swift]: This script will be called by all 4 bitrise workflows. The tests workflow (triggered by PR) creates the comment with the links (using a URL shortener) if it doesn't already exist. Each of the app workflows then updates the destination of the short links to point to the latest build result.

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/yz2zhb8) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fyz2zhb8)
teacher: [Link](https://rebrand.ly/lfrpum4) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Flfrpum4)
parent: [Link](https://rebrand.ly/iljwysd) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Filjwysd)

**Pull request merged**